src/imagej/convert.py

```diff
@@ -511,22 +511,24 @@ def index_img_to_roi_manager(
 
 
 def _get_contours(
-    ij: "jc.ImageJ", index_img: np.ndarray
+    ij: "jc.ImageJ", index_img: "jc.RandomAccessibleInterval"
 ) -> List["jc.DefaultWritablePolygon2D"]:
-    """
-    Compute contours from an index image.
+    """Compute contours from an index image.
+
+    Compute the contours from an index image by converting the image into
+    an ImgLabeling and applying the contour op on each region.
 
     :param ij: The ImageJ2 gateway (see imagej.init)
-    :param index_img: An ImgLib2 or NumPy index image
-    :return: A list of imglib2 2D poylgon ROIs
+    :param index_img: An RandomAccessibleInterval index image
+    :return: A list of DefaultWritablePolygon2D ROIs
```

Comment:
Please promise only a list of `WritablePolygon2D` ROIs, not the default implementation, which is supposed to be an internal detail.

Reply:
Done! eaa7e97d961540247bb585d53606411bf1af4e0e